test/e2e/view-offender-manager-overview.js

```diff
@@ -0,0 +1,25 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../helpers/data/aggregated-data-helper')
```

Comment:
We could rename this variable to something along the lines of `dataHelper` to make it more generic.